packages/phpstan-extensions/src/ErrorFormatter/SymplifyErrorFormatter.php

```diff
@@ -135,7 +135,7 @@ private function printSingleError(Error $error): void
 
         // ignored path
         $regexMessage = $this->regexMessage((string) $error->getMessage());
-        $itemMessage = sprintf(" - '%s'", $regexMessage);
+        $itemMessage = sprintf(' - "%s"', $regexMessage);
```

Comment:
Cases like these should be skipped. Or refactored to concat or use SymfonyStyle->list()

Reply:
fixed.